tests/core/config/startupFocus.js

```diff
@@ -0,0 +1,80 @@
+/* bender-tags: editor */
+// jscs:disable maximumLineLength
+/* bender-ckeditor-plugins: about,basicstyles,bidi,blockquote,clipboard,colorbutton,colordialog,div,enterkey,entities,find,flash,font,format,forms,horizontalrule,image,iframe,indent,justify,link,list,newpage,pagebreak,pastefromword,pastetext,removeformat,resize,toolbar,save,selectall,showblocks,smiley,sourcearea,specialchar,stylescombo,table,templates,undo,wysiwygarea */
+// jscs:enable maximumLineLength
+
+
+( function() {
+	'use strict';
+
+	function botInitializer( setup ) {
+		bender.editorBot.create( {
+			creator: 'replace',
+			name: setup.name,
+			config: Object.assign({
```

Comment:
`Object.assign` is not supported in some IEs, you may try [`CKEDITOR.tools.extend`](https://docs.ckeditor.com/ckeditor4/docs/#!/api/CKEDITOR.tools-method-extend) instead.